webpack/app/listing.coffee

```diff
@@ -94,6 +94,7 @@ class ListingController extends React.Component
     @listing_identifier = @root_el.dataset.listing_identifier
     @pagesize = parseInt @root_el.dataset.pagesize
     @review_states = @parse_json @root_el.dataset.review_states
+W    @default_review_state = @root_el.dataset.default_review_state
```

Comment:
Remove the `W` at the beginning of the line

Reply:
Oops, got this W in at some point after the build, apologies!